tutorials/btp-integration-suite-execute-batch-payload/btp-integration-suite-execute-batch-payload.md

```diff
@@ -0,0 +1,377 @@
+    		<dbTableName action="INSERT">
+    			<table> samplejdbctest</table>
+    			<access>
+    				<emp_id>1</emp_id>
+    				<emp_name hasQuot="Yes">v XXXXX</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement1>
+    	<insert_statement2>
+    		<dbTableName action="INSERT">
+    			<table>samplejdbctest</table>
+    			<access>
+    				<emp_id>2</emp_id>
+    				<emp_name hasQuot="Yes"> g28</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement2>
+    </root>
+    ```
+
+      <!-- border -->![Sample Playload](Payload.png)
+
+Notice that the JDBC adapter provides the content from the database in a specific format as described under [ Defining XML Documents for Message Protocol XML SQL Format](https://help.sap.com/docs/SAP_NETWEAVER_750/5cf7d2de571a45cc81f91261668b7361/2e96fd3f2d14e869e10000000a155106.html?version=7.5.4&locale=en-US)
+
+To check the Message Processing Log (MPL) of this integration flow from your tenant, go to **Monitoring** > **Overview** > **All Integration flows**. Choose the latest record with your artefact name.
+
+  <!-- border -->![Payload Processed](Payload-Processed.png)
+
+> Ensure to check the time stamp to determine which record to check.
+
+
+
+### Atomic Batch Operation
+
+Atomic Batch Operation doesn't allow an action to be performed on the payload if an error is encountered. To verify that, follow these steps:
+
+
+1. Copy the above payload and modify just one record and make it unique. For example, here we are making the **`emp_id`** (of second record) as 3. Thus, making that record unique, but the first record is duplicate. Send this query.
+
+    ```
+    <root>
+    	<insert_statement1>
+    		<dbTableName action="INSERT">
+    			<table> samplejdbctest</table>
+    			<access>
+    				<emp_id>1</emp_id>
+    				<emp_name hasQuot="Yes">v XXXXX</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement1>
+    	<insert_statement2>
+    		<dbTableName action="INSERT">
+    			<table>samplejdbctest</table>
+    			<access>
+    				<emp_id>3</emp_id>
+    				<emp_name hasQuot="Yes"> g28</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement2>
+    </root>
+    ```
+
+      <!-- border -->![Atomic Batch Operation](Atomic-Operation.png)
+
+     Postman displays error. To get more clarity, let's check the MPL.  
+    <!-- border -->![MPL for Atomic Batch Operation](MPL-Atomic.png)
+
+
+2. Let's confirm, whether unique record (with `emp_id` 3) is inserted in the database or not. To do this, uncheck the **Batch Mode** checkbox and **Deploy** your integration flow.
+
+    <!-- border -->![Batch Mode Off](Batch-Off.png)
+
+Then, **Send** the following query from Postman:
+
+`select * from samplejdbctest`
+
+  <!-- border -->![Atomic Batch Operation Verification](Verify-Atomic.png)
+
+This response shows that no new data has been inserted. Hence, verifying the Atomic operation.
+
+### Non-Atomic Batch Operation
```

Comment:
Please use sentence case

Reply:
Atomic is a special functionality of the product (JDBC adapter), which we are demonstrating here.